openc3-ruby/Dockerfile

```diff
@@ -73,6 +78,11 @@ RUN apk update \
   yaml-dev \
   && gem update --system 3.3.14 \
   && gem install rake \
+  # These need to be installed with the --platform flag to avoid errors on linux-musl
+  # See: https://github.com/protocolbuffers/protobuf/issues/16853#issuecomment-2583135716
+  # Should be fixed April 2025
```

Comment:
There's a chance this gets fixed earlier judging by the comments but I wouldn't hold my breath

Reply:
Its affecting everyone on alpine so I hope they fix it.